torch/onnx/_internal/io_adapter.py

```diff
@@ -103,6 +225,27 @@ def apply(
         return tuple(model_args) + tuple(model_kwargs.values()), {}
 
 
+class AppendPositionalsIntoArgsStep:
```

Comment:
I wonder if we should go specific and call `LiftParametersAndBuffersIntoArgsStep`.
This is fine too but maybe `AppendAdditionalArgsStep`?

Reply:
`LiftParametersAndBuffersIntoArgsStep` is more informative about the actual reason, I liked! will rename